src/screens/reader/ReaderScreen.js

```diff
@@ -392,16 +397,7 @@ const ChapterContent = ({ route, navigation }) => {
           theme={theme}
         />
         <GestureRecognizer
-          onSwipeRight={
-            swipeGestures &&
-            (!useWebViewForChapter || !wvUseNewSwipes) &&
-            navigateToPrevChapter
-          }
-          onSwipeLeft={
-            swipeGestures &&
-            (!useWebViewForChapter || !wvUseNewSwipes) &&
-            navigateToNextChapter
-          }
+          onSwipe={navigateToChapterBySwipe}
```

Comment:
`swipeGestures && (!useWebViewForChapter || !wvUseNewSwipes)` should be true for Swipe Navigation to be enabled.

Reply:
Why we cant use `swipeGestures` for `webview`. Also I dont know what `wvUseNewSwipes` does.
So only swipeGestures is enough imo

Reply:
So you should be unnecessary right now?. Cause we can swipe in Web mode without it

Reply:
Then we would also have to remove it from the setting, since we are combining them.

Reply:
okay, lemme try